lib/pdk/cli/validate.rb

```diff
@@ -17,6 +17,11 @@ module PDK::CLI
     flag :a, 'auto-correct', _('automatically correct problems (where possible)')
 
     run do |opts, args, _cmd|
+      if args == ['help']
```

Comment:
`validate` is "special" because the choice of validator conflicts with CRI's parsing of `validate help`. Try it, acceptance tests will fail.